### ISSUE TITLE
Device error when using pitch wheel

### ISSUE THREAD

**Issue by andreykaipov** [MiguelGuthridge/Novation-LaunchKey-Mk2-Script#45]
Hi - first off awesome extension! Still learning so I was just playing around with all the inputs and knobs and all that. When I touched the pitch wheel, all the pads lit up red and everything exploded.

It happens when I restart FL Studio and immediately touch the pitch wheel. Here's the output of both scripts:

[launchkey mk2 49 output.txt.txt](https://github.com/MiguelGuthridge/Novation-LaunchKey-Mk2-Script/files/5806244/launchkey.mk2.49.output.txt.txt)
[launchkey mk2 49 extension output.txt](https://github.com/MiguelGuthridge/Novation-LaunchKey-Mk2-Script/files/5806245/launchkey.mk2.49.extension.output.txt)

**Comment by andreykaipov** [comment]
Digging through it, I see there's [a comment that changing the pitch requires version 8](https://github.com/MiguelGuthridge/Novation-LaunchKey-Mk2-Script/blob/v2.1.0/pluginprocessors/processplugins.py#L85), and in fact yeah that's what it says on the [MIDI scripting page](https://www.image-line.com/fl-studio-learning/fl-studio-online-manual/html/midi_scripting.htm).

I guess I have to update to 20.8, don't I? In any case, I've already typed this up, and it might be a good idea to add that conditional back in so it doesn't crash the controller, so I'll leave this open if that's alright.

Again, thanks for this tool!

**Comment by andreykaipov** [comment]
Yeah adding the conditional back lets me bind it to something at least

**Comment by andreykaipov** [comment]
Yeah adding the conditional back lets me bind it to something else at least

**Comment by MaddyGuthridge** [comment]
That's interesting. I'm pretty sure I included a check to make sure that wouldn't happen, but I might have done it on a newer version branch. Either way, I've made a few changes to it now so it should work again in the next update (although that update will require FL 20.8 due to some other features). I'll have a quick look over that and see if I can pull the fix over to the main branch later today.
